src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, a) {
+      operation.call(this, a);
+
```

Comment:
The operate function assumes 'a' will be a number. There is no error checking to ensure that 'a' is a number. Add a check to ensure that 'a' is a number before using it.